benchmarks/src/commonMain/kotlin/PeekBenchmark.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package kotlinx.io.benchmarks
+
+import kotlinx.io.*
+import kotlinx.benchmark.*
+
+@State(Scope.Benchmark)
+abstract class PeekBenchmark {
+    protected val buffer = Buffer()
+
+    @Param("0", "8191")
```

Comment:
8191 - is a magic value?

Reply:
It's an index of the last byte within the first segment. The idea was to hit segments' border in multi-byte benchmarks.
I'll add a comment regarding its meaning.